mautic/mautic
# Fix set import template to send custom email If use legacy builder

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... https://github.com/mautic/mautic/issues/10604



#### Description:

If GrapesJs builder is disabled. Then it's not possible import template in send email to contact form. This PR fixed it



#### Steps to test this PR:

T


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester)). Require `php bin/console mautic:assets:generate` on production
2. Disable GrapesJs 
3. Refresh browser
4. Go to contact detail page
5. Try send email with import template of existing email




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10897"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
